src/session_impl.cpp

```diff
@@ -473,6 +473,15 @@ namespace aux {
 			m_alerts.set_alert_mask(pack.get_int(settings_pack::alert_mask));
 		}
 
+		if (!pack.has_val(settings_pack::enable_dht)
+			|| pack.get_bool(settings_pack::enable_dht) == true)
+		{
+			// if the DHT is enabled on startup, there's currently no way to add
+			// router nodes early enough for them to be used by the DHT bootstrap
+			// add the default libtorrent router
+			add_dht_router(std::pair<std::string, int>("dht.libtorrent.org", 25401));
```

Comment:
yeah, I intend to do that. I think there may be a way to do it in RC_1_1 without breaking ABI even.. I believe the interaction with the session is via the settings_pack, and not aux::session_settings. That means it's just a map of settings-indices -> values. i.e. it can be extended without altering the ABI